tests/integ/test_stored_procedure.py

```diff
@@ -689,6 +694,115 @@ def _(
     assert "Pandas stored procedure is not supported" in str(ex_info)
 
 
+@pytest.mark.parametrize("is_permanent", [True, False])
+@pytest.mark.parametrize("anonymous", [True, False])
+@pytest.mark.parametrize(
+    "ret_type",
+    [
+        StructType([]),
+        StructType(
+            [
+                StructField("a", StringType()),
+                StructField("b", StringType()),
+                StructField("c", StringType()),
+            ]
+        ),
+    ],
+)
+@pytest.mark.skipif(
+    IS_IN_STORED_PROC,
+    reason="Named temporary procedure is not supported in stored proc",
+)
+def test_table_sproc(session, is_permanent, anonymous, ret_type):
```

Comment:
can we add a test for session.call, when the table stored proc doesn't exist, because the inferred type from arguments doesn't match with the original arg type of stored proc?

Reply:
added. I hope this is what you were expecting. see `test_negative_table_sproc`

Reply:
yes, thanks